shared/tracker/index.js

```diff
@@ -144,7 +93,18 @@ export default connect(
   }),
   dispatch => {
     return bindActionCreators(trackerActions, dispatch)
-  })(Tracker)
+  },
+  (stateProps, dispatchProps, ownProps) => ({
+    ...ownProps,
+    ...stateProps,
+    ...dispatchProps,
+    onClose: () => dispatchProps.onClose(ownProps.username),
```

Comment:
actually i forget the dispatch method also takes in an optional ownProps so i cleaned that up here: https://github.com/keybase/client/pull/3162/files#diff-8b1f247864695c72166680e06573f56bR94